README.md

```diff
@@ -18,16 +18,29 @@ supported environments.
 The reference examples should reflect the desired format.  The IDE configurations may not support or respect this, 
 which is okay while we're developing the format and auto-formatters.
 
-## Installation
+# Goals, Intent and Working Agreement
 
-### IntelliJ IDEA
+## Goals
+We want to present a unified approach to code styling across our entire suite of development efforts, automating away
+the need to respond to deviations during integration of code from a large group of developers, both within and outside
+of the Sonatype Organization.
+
+## Intent
+All of the development environments used by developers contributing to Sonatype code should be applying the same agreed
+upon formatting rules consistently, so that we don't spend needless time aligning these rules along with the more
+substantial (and important!) logical changes we make on a daily basis.
+
+## Working Agreement
+- Any change to existing rules should be discussed and agreed upon by the larger group before implementation.
+- Any noted deviation between development environments should result in the scheduling of work to correctly align with 

```

Comment:
This seems to preclude Improvement Day work or someone noticing an issue while working with a dev evironment and submitting a PR.  Am I reading that right?  Is that the intent?

Reply:
The intent is to ensure that we are aligned across all supported platforms. At least from my perspective I wouldn't expect that anyone should spend ID day working on what is absolutely an important and organization supported initiative. This is not innovation - it's essential work to ensure we are all utilizing the same rule set, and I'd fully expect teams to make room in their schedule for addressing the problem the same as any other "bug".